permafrost/tests.py

```diff
@@ -23,7 +23,8 @@
     PermafrostRole,
     get_current_site,
     CATEGORIES,
-    get_all_perms_for_all_categories
+    get_all_perms_for_all_categories,
+    PERMAFROST_EXCLUDED_ROLES
```

Comment:
@fahzee1 don't import here
should come form

from django.conf import settings